.github/workflows/ci.yml

```diff
@@ -43,8 +48,9 @@ jobs:
           source .tinyenv
           echo "GIT_BRANCH=$GIT_BRANCH" >> $GITHUB_ENV
           echo "_ENV_FLAGS=$_ENV_FLAGS" >> $GITHUB_ENV
+          echo "_NORMALIZED_ENV_NAME=$_NORMALIZED_ENV_NAME" >> $GITHUB_ENV
       - name: Install Tinybird CLI
-        run: pip install tinybird-cli
+        run: pip install tinybird-cli==1.0.1.dev1
```

Comment:
You can also just use `pip install tinybird-cli --pre`

Reply:
💥